rllib/utils/replay_buffers/tests/test_episode_replay_buffer.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+
+import numpy as np
+
+from ray.rllib.policy.sample_batch import SampleBatch, concat_samples
+from ray.rllib.utils.replay_buffers.episode_replay_buffer import EpisodeReplayBuffer
+from ray.rllib.utils.replay_buffers.utils import ReplayBufferEpisode
+
+
+class TestEpisodeReplayBuffer(unittest.TestCase):
```

Comment:
Can we include this test in rllib's BUILD?
Also, I tried running it and it does not run because the imports are off.

Reply:
Oh, forgot! :)

Reply:
done.
Adding some more test cases to this, though ...